torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -12,16 +12,15 @@
 )
 
 
-def empty(
-        sharding_spec: ShardingSpec,
-        *size,
-        dtype=None,
-        layout=torch.strided,
-        requires_grad=False,
-        pin_memory=False,
-        memory_format=torch.contiguous_format,
-        process_group=None,
-        init_rrefs=False):
+def empty(sharding_spec: ShardingSpec,
```

Comment:
nit: It is easier to review code if formatting changes are in a separate PR.

Reply:
Since empty is the only one which does not align with the format of rest. Maybe just keep it since it's a tiny change?